src/icon-buttons/bold-button.component.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useDocumentExecCommand } from '../use-document-exec-command.hook'
+import { useDocumentQueryCommandState } from '../use-document-query-command-state.hook'
+import IconButton from './icon-button.component'
+import BoldIcon from './bold-icon.component'
+
+export function BoldButton(props) {
```

Comment:
I think this entire component should be deleted - let the user of bandicoot use the hooks and IconButton/BoldIcon components